configs/compute/1gpu.cfg

```diff
@@ -21,5 +21,8 @@ nnRandomize = true
 deviceToUseThread0 = 0
 
 useFP16 = true
-# We only set NHWC for CUDA because TensorRT requires useNHWC == false.
-cudaUseNHWC = true
+# The TorchScript backend requires NCHW.
```

Comment:
Is this harmless for non-ViT models (b/c we're not using TensorRT backend) or could it have some performance implications?

Reply:
yeah, I should've benchmarked this. Seems like this makes CUDA-backend models ~30% slower, so I'll revert this and leave a comment that this param should be toggled for TorchScript-backend models. Might be good for me to just override NHWC to be false in the C++ code for TorchScript models, but I notice the TensorRT backend also just throws an error if NHWC is set incorrectly so I don't feel too bad about not implementing forceful overriding